_browser/blockstack_storage.md

```diff
@@ -630,40 +577,25 @@ process URL paths that contain the `.` (dot) character for example,
      documentation](http://blockstack.github.io/blockstack.js/#getfile) for
      details.
 
-5. Add the following block to `fetchData()` right after the call to `lookupProfile(username)... catch((error)=>{..}` block:
+5. Add the following block to `fetchData()` right after the call to `setUsername(username)` block:
```

Comment:
Why is this another step? Seems to add confusion and I don't understand the code better

Reply:
I agree that it doesn't make much sense as a separate step.